beeswax/augment/augmentor.proto

```diff
@@ -23,20 +24,34 @@ message AugmentorRequest {
 //
 // Also custom bidding agents can receive this data through the augmentor_data field in
 // the BidRequestExtension.
+// Next id: 3
 message AugmentorResponse {
 
+  // Next id: 4
   message Segment {
     // The Id of an augmentor segment.
     // This field is primarily used for line item targeting.
     // For targeting to work with the segment ids, it is necessary
     // to register the segment ids with Beeswax via the Buzz Segments API.
     optional string id = 1;
 
-    // This field can be used to pass additional information about the segment
-    // that Stinger will forward to the bidding agent.
-    optional string value = 2;
+    // This is a deprecated field which is not used any more.
+    optional string deprecated_string_value = 2 [deprecated = true];
```

Comment:
remove `deprecated_string_value` and `value` in this update

Reply:
I meant we should undo the change here and keep the old string value in the proto as is